scripts/create-symlinks.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
You might want to use the unofficial [strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/).